release.config.js

```diff
@@ -16,7 +16,7 @@ module.exports = {
     [
       '@semantic-release/github',
       {
-        assets: ['CHANGELOG.md'],
+        assets: ['CHANGELOG.md', 'swagger-spec.json'],
```

Comment:
Theoretically this should do the trick, but I was not able to test this locally. Will have to merge a PR to get this tested. Any other way to do this @polymath-eric @VictorVicente ?